src/Executables/Game/GameServer.cs

```diff
@@ -106,6 +109,10 @@ await Task.WhenAll(
                 _skillManager.LoadAsync(stoppingToken)
             );
 
+            // Drop all pre-existing caches
+            _logger.LogInformation("Drop all caches");
+            await _cacheManager.FlushAll();
```

Comment:
Not sure of this. We may consider another approach.

Reply:
Why? What are you trying to accomplish with that?
Think distributed: What if you spin up another server: It would flush the distributed cache of another instance too!

Reply:
The main purpose is invalidate all tokens before the server ready to accept connections. Actually in current structure and use of redis, distributed cache is not possible because of items and tokens which are just integers with some duplicable prefix. Maybe we should reconsider/refactor the approach of cache with increased usage of such commands like `SELECT` and `BGSAVE`.